src/core/cgroup.c

```diff
@@ -293,8 +293,13 @@ static int whitelist_device(const char *path, const char *node, const char *acc)
         assert(acc);
 
         if (stat(node, &st) < 0) {
-                log_warning("Couldn't stat device %s", node);
-                return -errno;
+                /* path starting with "-" must be silently ignored */
+                if (startswith(node, "-"))
+                        return 0;
+                else {
+                        log_warning("Couldn't stat device %s", node);
+                        return -errno;
```

Comment:
While we are at it we should probably merge the two lines to become "return log_warning_erro(errno, …"...